src/tests/mongodb.utils.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Automatically reads MONGODB_URI and replaces the connected database with
+ * the testing database (default name: 'test') to prevent accidental purging
+ * of data in the production/development/stage database.
+ *
+ * @example MONGODB_URI = 'mongodb://localhost:27017/development'
+ * @returns 'mongodb://localhost:27017/test'
+ */
+export const useTestingDatabase = () => {
+  if (process.env.MONGODB_URI)
+    return process.env.MONGODB_URI.replace(/\/[^/]+$/, '/test');
```

Comment:
MONGODB_URI should always exists in local development, but it might be pointing to other databases (i.e. `/development`, `/stage`, ...). 

This function will simply return the MONGODB_URI but points it to `/test` instead.